bin/l10n-sync.js

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This script will sync the l10 branch with main branch
+// Run with -y to automatically skip the prompts.
```

Comment:
Can you add one more line about how to run it? Is it `node bin/l10n-sync.js` from the root? Do we need to be at the root or can we be anywhere in the project? (I think we can be anywhere indeed, because we only do branch operations).

Reply:
Yeah, I usually run it with `node bin/l10n-sync.js` but you don't necessarily be at the root directory. Added a comment to explain that.